src/sentry/templates/sentry/idp_email_verified.html

```diff
@@ -0,0 +1,14 @@
+{% extends "sentry/emails/base.html" %}
+
+{% load i18n %}
+
+{% block main %}
+    <h3>Thank you!</h3>
+    <p>
+        Your email has been confirmed. Click the button below to log in to your Sentry account.
+    </p>
+
+    <form class="form-stacked" action="" method="post"> {% csrf_token %}
+        <a href="{{ url }}" class="btn" type="submit">Continue</a>
+    </form>
```

Comment:
Why is there a link inside a form? I don't think you'll have much luck with a form inside an email either.